openpype/hosts/maya/plugins/publish/collect_render.py

```diff
@@ -405,3 +420,45 @@ def get_render_attribute(self, attr, layer):
         return lib.get_attr_in_layer(
             "defaultRenderGlobals.{}".format(attr), layer=layer
         )
+
+    @staticmethod
+    def _collect_deadline_url(render_instance):
```

Comment:
Can this be in deadline collector? This adds deadline logic to Maya's plugin.

Reply:
Done